preview/Win7Msix/Win7MSIXInstallerLib/inc/Win7MSIXInstallerActions.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "DllExport.hpp"
+#include "IMsixRequest.hpp"
+#include <AppxPackaging.hpp>
+#include <vector>
+
+extern "C" DLLEXPORT HRESULT Win7MsixInstallerLib_CreateAddPackageRequest(std::wstring packageFilePath, MSIX_VALIDATION_OPTION validationOption, Win7MsixInstallerLib::IMsixRequest** outInstance);

```

Comment:
>Win7MsixInstallerLib_CreateAddPackageRequest [](start = 29, length = 44)

@rudyhuyn - Thanks for sharing this PR. I have sent you an email with some details about the API surface to be exposed from this lib that will meet more customer scenarios and requirements. Let's follow up in email and we can proceed accordingly with this PR. Thanks!

Reply:
Done!